doc/radosgw/encryption.rst

```diff
@@ -48,7 +48,7 @@ SSE-S3
 ======
 
 This makes key management invisible to the user.  They are still stored
-in vault, but they are automatically created and deleted by Ceph. and
+in a vault, but they are automatically created and deleted by Ceph and
```

Comment:
I think maybe the even better fix might be

"in Vault, but they are .."

Because on line 42 a service named `Vault` is referenced, and moreover on line 58 below we have "only integration with Vault is implemented".

So I think here `Vault` is a proper noun and should be capitalized.


Also, please review https://github.com/ceph/ceph/blob/main/doc/start/documenting-ceph.rst

There are a number of conventions and practices that the Ceph project likes to follow.  For this PR (thanks!) that includes:

* Edit the PR description to check off one box in each section, there's a test that I'll unblock that will fail without this
* Change the PR and Commit titles to be of the form "doc/radosgw: Fix punctuation and grammar in encryption.rst"

If you need some help doing these, please reply here and I'll be in touch.

Reply:
Thanks for the suggestions and direction. Let me know if any other changes are needed.

Reply:
Looks good to me. Thanks for contributing!

Reply:
You'll want to edit the Checklist again and check "No impact that needs to be tracked" and "No tests".
The checklist is geared toward code submissions, so for docs we just breeze through it.